packages/transport/src/transports/bridge.ts

```diff
@@ -135,20 +135,13 @@ export class BridgeTransport extends AbstractTransport {
 
     private async listenLoop() {
         while (!this.stopped) {
-            const listenTimestamp = Date.now();
-
             const response = await this.post('/listen', {
                 body: this.descriptors,
                 signal: this.abortController.signal,
             });
 
             if (!response.success) {
-                const time = Date.now() - listenTimestamp;
-                if (time <= 1100) {
-                    this.emit(TRANSPORT.ERROR, response.error);
-                    break;
-                }
-                await resolveAfter(1000);
```

Comment:
I had to get rid of this. For some reason, toggling from new to legacy did not emit transport error. Now both variants work (when we talk about emitting transport-error followed by transport-start after clicking the "toggle" button that I am adding here). cc @marekrjpolak